cmd/abci-cli/abci-cli.go

```diff
@@ -382,18 +380,70 @@ func cmdConsole(cmd *cobra.Command, args []string) error {
 		}
 
 		pArgs := persistentArgs(line)
-		out, err := exec.Command(pArgs[0], pArgs[1:]...).Output() // nolint: gas
-		if err != nil {
+		if err := muxOnCommands(cmd, pArgs); err != nil {
 			return err
 		}
-		fmt.Println(string(out))
 	}
 	return nil
 }
 
+var (
+	errBadPersistentArgs = errors.New("expecting persistent args of the form: abci-cli [command] <...>")
```

Comment:
why a var if it only used in one place?

Reply:
Great question, that's a vestige of the former code. I'll remove it.